101-backup-summary-report/prereqs/.settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
remove this file and ensure you sample works when using separate resourceGroups (since it's not required they are the same)